src/web3/saga.ts

```diff
@@ -258,6 +304,56 @@ export function* assignAccountFromPrivateKey(privateKey: string, mnemonic: strin
   }
 }
 
+// TODO
+// That should be replace with a real flow to input the email and verify
+async function createFakeAccount() {
+  const { userId } = await createUser({
+    email: `test-${uuidv4()}@test.usecapsule.com`,
+  })
+  Logger.debug('userId', userId)
+  await AsyncStorage.setItem(USER_ID_TAG, userId)
+
+  // That is a workaround to simulate verification of test users
+  await verifyEmail(userId, { verificationCode: '123456' })
+  Logger.debug('Verified!')
+}
+
+export function* createAndAssignCapsuleAccount() {
+  try {
+    yield call(createFakeAccount)
+    Logger.debug(TAG + '@createAndAssignCapsuleAccount', 'Attempting to create wallet')
+    const wallet: CapsuleWallet = yield call(getWallet)
+    Logger.debug(TAG + '@createAndAssignCapsuleAccount', 'Got wallet')
+    let account = ''
```

Comment:
Is this for type safety, or am I missing something?

Reply:
Probably some left-over! @codyborn